### PR TITLE
Update Android Docs 

### PR DESCRIPTION
**Pull request by PeloWriter**
The work for this PR is listed in this [issue](https://github.com/getsentry/sentry-docs/issues/3237).

### CODE REVIEW

File: src/includes/getting-started-primer/android.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<Note>
+
+*Sentry's Android SDK reports an error automatically whenever a thrown exception goes uncaught in your application causing the application to crash.*
+
+*The SDK builds a crash report that persists to disk and tries to send the report right after the crash; however, because the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again. This process for sending a report is true if there is a fatal error in your native code as well as for the NDK. In addition, the NDK is not only catching unhandled exceptions but is also set as a signal handler to react to signals from the OS.*
+
+</Note>
+
+**Features:**
```
Comment (marandaneto) [inline src/includes/getting-started-primer/android.mdx:9]: I'd add attachments, user feedback and ANR detection

**Comment by marandaneto** [comment]
@PeloWriter 

auto-session-tracking - missing for android, ios, RN (or fallback to JS?)
cache-directory-path - missing for android, do iOS and RN offer that?

I just spotted these common options missing

**Comment by jennmueng** [comment]
> @PeloWriter
> 
> auto-session-tracking - missing for android, ios, RN (or fallback to JS?) @philipphofmann @jennmueng
> cache-directory-path - missing for android, do iOS and RN offer that? @philipphofmann @jennmueng
> 
> I just spotted these common options missing

For `auto-session-tracking`, on RN it is `enableAutoSessionTracking` not `autoSessionTracking`, I think it falls back to JS as it's being shown @PeloWriter 
For `cache-directory-path` RN does not offer it, the user would need to init the Android SDK manually to do so.

File: src/includes/getting-started-primer/android.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Automatic native crash error tracking, including Java, Kotlin, C, and C++ code
+- Events [enriched](/platforms/android/enriching-events/context/) with device data
+- [Breadcrumbs automatically](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) captured
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
+- [Attachments](/platforms/android/enriching-events/attachments/) enrich your event by storing additional files, such as config or log files.
+- [User Feedback](/platforms/android/enriching-events/user-feedback/) provides the abiity to collect user information when an event occurs.
+- [Performance Monitoring](/product/performance/) creates transactions
+- Application Not Responding (ANR) reported if the application is blocked for more than four seconds.
```
Comment (marandaneto) [inline src/includes/getting-started-primer/android.mdx:18]:
```suggestion
- Application Not Responding (ANR) reported if the application is blocked for more than five seconds.
```

**Comment by PeloWriter** [comment]
@jennmueng - I added your comment to https://github.com/getsentry/sentry-docs/issues/2985. Thanks.

File: src/includes/configuration/auto-session-tracking/android.mdx [hunk 3]
```diff
@@ -51,3 +51,17 @@ SentryAndroid.init(this) { options ->
...
 }
 ```
+### AndroidX Support
+
+Sentry uses the AndroidX libraries for detecting when the application is either in the background or in the foreground. This is necessary for having an accurate result across all the Android OS versions.
+
+We check at runtime for availability, so if you're not using AndroidX libraries, you can remove them from Sentry's transitive dependencies.
+
+```Groovy
+implementation ('io.sentry:sentry-android:4.3.0') {
```
Comment (marandaneto) [inline src/includes/configuration/auto-session-tracking/android.mdx:61]:
```suggestion
implementation ('io.sentry:sentry-android:{{ packages.version('sentry.java', '4.2.0') }}') {
```

File: src/platforms/android/common/proguard.mdx [hunk 4]
```diff
@@ -76,5 +79,23 @@ sentry {
 }
 ```
 
-And that's it! Now when you build your app, the plugin will upload the ProGuard/R8 mappings,
-source bundle, and native symbols, as you configured them to Sentry.
+Now when you build your app, the plugin will upload the ProGuard/R8 mappings, source bundle, and native symbols as you configured them to Sentry.
+
+### Multi-Dex Support
```
Comment (marandaneto) [inline src/platforms/android/common/proguard.mdx:84]: I don't think this should be under this section.

the configuration is similar to the proguard way, but it's not related to it.

I also think we don't have a suitable section for it, it's more a general advanced config, or caveats or something like that

File: src/platforms/common/enriching-events/scopes.mdx [hunk 5]
```diff
@@ -63,13 +63,11 @@ then merge the event with the topmost scope's data.
...
 
-The most useful operation when working with scopes is the `configure-scope` function.
-It can be used to reconfigure the current scope. This for instance can be used to
-add custom tags or to inform sentry about the currently authenticated user.
+The most useful operation when working with scopes is the `configure-scope` function. It can be used to reconfigure the current scope. This for instance can be used to add custom tags or to inform Sentry about the currently authenticated user.
 
 <PlatformContent includePath="configure-scope" />
 
-This can also be applied when unsetting a user at logout:
+You can also be apply this configuration when unsetting a user at logout:
```
Comment (marandaneto) [inline src/platforms/common/enriching-events/scopes.mdx:70]:
```suggestion
You can also apply this configuration when unsetting a user at logout:
```

File: src/platforms/android/common/proguard.mdx (hunk 4 above)
Comment (marandaneto) [reply src/platforms/android/common/proguard.mdx:84]: maybe we create a section under platforms/android/configuration/ ?

**Comment by philipphofmann** [comment]
> cache-directory-path - missing for android, do iOS and RN offer that?

We don't have this for iOS.

File: src/platforms/android/common/proguard.mdx (hunk 4 above)
Comment (PeloWriter) [reply src/platforms/android/common/proguard.mdx:84]: That was my attempt :) But, yes, I'll make it its own page under Config. Thanks for double checking, @marandaneto

**Review by marandaneto — APPROVED** [review]
looks great @PeloWriter thanks
LGTM

File: src/platforms/android/manual-init.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+title: Manual Initialization
```
Comment (marandaneto) [inline src/platforms/android/manual-init.mdx:2]: just a nit, so feel free to ignore if you think this isn't relevant.

this page could go under `platforms/android/configuration/` as it's not a feature or anything, just a configuration you can do, most of the people don't, I believe.

Comment (PeloWriter) [reply src/platforms/android/manual-init.mdx:2]: I agree, upon reflection and deeper reading of what we're covering here. Thanks.

**Pull request merged**